cluster/vagrant/provision-minion.sh

```diff
@@ -72,16 +72,17 @@ EOF
 # See: https://github.com/mitchellh/vagrant/issues/2430
 hostnamectl set-hostname ${MINION_NAME}
 
-# Workaround to vagrant inability to guess interface naming sequence
-# Tell system to abandon the new naming scheme and use eth* instead
-rm -f /etc/sysconfig/network-scripts/ifcfg-enp0s3
```

Comment:
I still think this is required to be removed when running on Virtual Box.

If VMWare does not have this problem, can you pass the virtualization provider as an env var into the provision scripts, and keep this file only when running on VMWare?